### PR TITLE
minor fixes in project files

### PR DESCRIPTION
**Pull request by stephanenicolas**
Just a few things here and there.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ Release procedure for dependency-analysis-android-gradle-plugin
 1. Bump version number to next stable version (use semantic versioning: x.y.z)
 1. Update README if needed
 1. git commit -am "Prepare for release x.y.z." 
-1. git tag -a vx.y.z -m "Version x.y.z"
+1. git tag -a x.y.z -m "Version x.y.z"
```
Comment (autonomousapps) [inline RELEASING.md]: The `v` is intentional ;-)

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <a alt="Latest version" href="https://plugins.gradle.org/plugin/com.autonomousapps.dependency-analysis">
-<img src="https://img.shields.io/maven-metadata/v.svg?label=gradle&metadataUrl=https%3A%2F%2Fplugins.gradle.org%2Fm2%2Fcom%2Fautonomousapps%2Fdependency-analysis%2Fcom.autonomousapps.dependency-analysis.gradle.plugin%2Fmaven-metadata.xml"/></a>
+<img src="https://img.shields.io/maven-metadata/v.svg?label=plugin%20version&metadataUrl=https%3A%2F%2Fplugins.gradle.org%2Fm2%2Fcom%2Fautonomousapps%2Fdependency-analysis%2Fcom.autonomousapps.dependency-analysis.gradle.plugin%2Fmaven-metadata.xml"/></a>
```
Comment (autonomousapps) [inline README.md]: :+1:

File: RELEASING.md [hunk 3]
```diff
@@ -4,11 +4,11 @@ Release procedure for dependency-analysis-android-gradle-plugin
...
 1. Execute `./gradlew publishPlugins` to release to the [Plugin Portal](https://plugins.gradle.org/plugin/com.autonomousapps.dependency-analysis)
 This will also run the tests.
-1. Update `build.gradle.kts` to next snapshot version (x.y.z-SNAPSHOT)
-1. Bump `latestRelease` to stable version just published.
+1. Update `build.gradle.kts` to next snapshot version (x.y.z+1-SNAPSHOT)
```
Comment (autonomousapps) [inline RELEASING.md]: What's the convention you're proposing here? What will iterative snapshots look like?

Comment (stephanenicolas) [reply RELEASING.md]: There is no special convention, x.y.z was released in the previous step, so the next snapshot cannot be named x.y.z-snapshot, it's an increased version number, and the simplest way to express it is to increase z. If it's not clear the remove the whole parentheses.

**Comment by autonomousapps** [comment]
I was specifically referring to the `+1`.

File: RELEASING.md (hunk 3 above)
Comment (stephanenicolas) [reply RELEASING.md]: For the '+1', it's just about saying that we should not forget that while adding `-SNAPSHOT`, we should also not forget to increase the version number. Let me know if you find a better / more clear way to say it.

**Comment by stephanenicolas** [comment]
I am not sure to understand well if the PR is still blocked. The first comment was addressed too.

File: RELEASING.md [hunk 4]
```diff
@@ -4,11 +4,11 @@ Release procedure for dependency-analysis-android-gradle-plugin
 1. Bump version number to next stable version (use semantic versioning: x.y.z)
 1. Update README if needed
 1. git commit -am "Prepare for release x.y.z." 
-1. git tag -a vx.y.z -m "Version x.y.z"
+1. git tag -a -v x.y.z -m "Version x.y.z"
```
Comment (autonomousapps) [inline RELEASING.md]: This line should be unchanged. The tag is `-a vx.y.z`, not `-a -v x.y.z`.

**Comment by autonomousapps** [comment]
Manually changed the label in the readme just now. Closing this PR.

**Pull request closed without merging**